src/NativeOpentokReactNative.ts

```diff
@@ -30,10 +98,18 @@ export type SessionErrorEvent = {
 };
 
 export interface Spec extends TurboModule {
+  readonly onArchiveStarted: EventEmitter<ArchiveEvent>;
+  readonly onArchiveStopped: EventEmitter<ArchiveEvent>;
+  readonly onConnectionCreated: EventEmitter<ConnectionEvent>;
```

Comment:
Is it intentional to send ConnectionEvent for onConnectionCreated and onConnectionDestroyed?

Reply:
Yes. The existing TypeScript definitions have two event types, but they are exactly the same: https://github.com/opentok/opentok-react-native/blob/develop/%40types/index.d.ts#L20-L26.